flexmeasures/api/v3_0/assets.py

```diff
@@ -1062,3 +1062,100 @@ def trigger_schedule(
         response = dict(schedule=job.id)
         d, s = request_processed()
         return dict(**response, **d), s
+
+    @route("/<id>/kpis", methods=["GET"])
+    @use_kwargs(
+        {
+            "asset": AssetIdField(
+                data_key="id", status_if_not_found=HTTPStatus.NOT_FOUND
+            )
+        },
+        location="path",
+    )
+    @use_kwargs(
+        {
+            "start": AwareDateTimeField(required=True),
+            "end": AwareDateTimeField(required=True),
+        },
+        location="query",
+    )
+    def get_kpis(self, id: int, asset: GenericAsset, start, end):
+        from flexmeasures.data.services.sensors import get_sensor_stats
```

Comment:
This weirdly-placed import breaks the documentation of this endpoint. 

I recommend moving it to the top.